data/stoa0292b/stoa001/stoa0292b.stoa001.opp-lat1.xml

```diff
@@ -2,7 +2,9 @@
 <teiHeader>
     <fileDesc>
       <titleStmt>
-        <title>Corpus Scriptorum Ecclesiasticorum Latinorum (CSEL)</title>
+        <title>Alethia</title>
```

Comment:
The title is marked as lat above.

Reply:
Hi @lcerrato I will fix the title and it is Alethia not Aletheia.